### ISSUE TITLE
Cleanup utilities

### ISSUE THREAD

**Issue by funnym0nk3y** [dendronhq/dendron#650]
## Please Select if your Request is Either something new or an Enhancement ##

- [ ] Enhancement of an existing Feature.
- [x] Request of a new feature.

## Please select the area your request applys to. (Multiple selections are Possible) ##

- [ ] Onboard. Initial vault setup and import/export pods 
- [ ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [x] Misc (Choose this if your not sure)

## Is your feature request related to a problem? Please describe ##

Restructuring the knowledge graph as it is necessary with progress of lectures and research leaves a lot of unneeded stubs and assets. That leads to clutter. 

## Describe the solution you'd like ##

I'd like to have a few tools/functions that allow to declutter the workspace, e.g. delete all unlinked files (images) and empty stubs as they are not needed anymore. So that each file in the vault brings some information to the table.

**Comment by kpathakota** [comment]
@funnym0nk3y - have you tried the `Dendron: Doctor` to delete stubs? IF that works for you - does this issue now mainly ask for deleting unlinked files?

**Comment by funnym0nk3y** [comment]
Yes, this issue was mainly created to request deletion of unlinked assets. The sub cleanup idea just came when writing this issue.
